crates/sui-network/build.rs

```diff
@@ -81,7 +81,7 @@ fn main() -> Result<()> {
                 .name("get_system_state_object")
                 .route_name("GetSystemStateObject")
                 .input_type("sui_types::messages::SystemStateRequest")
-                .output_type("sui_types::sui_system_state::SuiSystemState")
+                .output_type("sui_types::sui_system_state::SuiSystemStateInnerBenchmark")
```

Comment:
Wondering why we use `SuiSystemStateInnerBenchmark` here. Is this only for benchmarking?

Reply:
Correct. This is used only for benchmarking today